File: server/DataCore/invw.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace inv19.models { 
	/* invw -  Наличие */ 

 public class  invw_info { // Наличие
	 public System.Guid  invw_infoId{ get; set; } // Идентификатор (первичный ключ)
	public System.Guid  theStore { get; set; } //Склад
	
	public System.Guid  locationid { get; set; } //Стеллаж
	
	public System.Guid  cellid { get; set; } //Ячейка

	public System.Guid  storepartid { get; set; } //Запчасть
	
	[Required]
	public double  Qty{ get; set; } // Количество
	[Required]
	public string  RFID{ get; set; } // Метка RFID
 }
}

File: client/src/app/invw.ts
import { enums } from './enums';

export namespace invw { 
	/* invw -  Наличие */ 

 export interface   invw_info { // Наличие
	invw_infoId:string; // Primary key field
	theStore:string; //Склад -> invd_store
	locationid:string; //Стеллаж -> invwh_loc
	cellid:string; //Ячейка -> invwh_cell
	storepartid:string; //Запчасть -> invp_data
	qty:Number; // Количество
	rFID:string; // Метка RFID
	// add dereference fields 
	theStore_name :string; // dereference for invd_store
	locationid_name :string; // dereference for invwh_loc
	cellid_name :string; // dereference for invwh_cell
	storepartid_name :string; // dereference for invp_data
 }
}